TWLight/applications/tests.py

```diff
@@ -1840,23 +1840,14 @@ def _base_test_deleted_object_visibility(self, url, view, queryset):
                 escape(obj.__str__()), response.render().content.decode("utf-8")
             )
 
-    def test_list_object_visibility(self):
-        url = reverse("applications:list")
-        queryset = Application.objects.filter(
-            status__in=[Application.PENDING, Application.QUESTION]
-        )
-        self._base_test_deleted_object_visibility(
-            url, views.ListApplicationsView, queryset
-        )
```

Comment:
I'm not actually seeing where else this queryset is being run against `_base_test_deleted_object_visibility`; this looks to me like we'd be losing test coverage.

Reply:
yes you were right I mistakenly deleted that thinking its duplicate, I have rewrote it now

Reply:
Thanks! These method names are a big improvement in descriptiveness in addition to removing the naming collisions.